nextcord/application_command.py

```diff
@@ -1815,6 +1815,8 @@ def __init__(
             Callback to make the application command from, and to run when the application command is called.
         guild_ids: Iterable[:class:`int`]
             An iterable list/set/whatever of guild ID's that the application command should register to.
+            If not passed and :attr:`Client.default_guild_ids` is set, then those default guild ids will
+            be used instead. If both of those are unset, then the command will be a global command. Defaults to ``MISSING``.
```

Comment:
I don't think it should say "Defaults to MISSING" anywhere in the docs. To the user of lib, this just means it's not passed.


```suggestion
            be used instead. If both of those are unset, then the command will be a global command.
```